recipes/pygama/recipe.yaml

```diff
@@ -0,0 +1,87 @@
+schema_version: 1
+
+context:
+  name: pygama
+  version: "2.2.3"
+  # pylegendmeta is Python 3.10 to get around a selector issue,
+  # so use Python 3.10 even though source is 3.9+.
+  python_min: "3.10"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/pygama-${{ version }}.tar.gz
+  sha256: bf04bf815bfdcb749180d8deba52238c094672a9e538e46fae94e0bd97284cc0
+
+build:
+  number: 0
+  noarch: python
```

Comment:
Windows fails as expected with

```
Error:   × Test failed: failed to setup test environment: Cannot solve the request
  │ because of: pygama ==2.2.3 pyh54f61e6_0 cannot be installed because there
  │ are no viable options:
  │ └─ pygama 2.2.3 would require
  │    └─ legend-pydataobj >=1.14, which cannot be installed because there are
  │ no viable options:
  │       └─ legend-pydataobj 1.14.4 would require
  │          └─ __unix *, for which no candidates were found.
```

given that we decided to add the `__unix` virtual package in https://github.com/conda-forge/staged-recipes/pull/30539.